pyproject.toml

```diff
@@ -176,6 +190,41 @@ known-first-party = ["tephi"]
     "ERA001", # Has commented out code
 ]
 
+[tool.tox]
```

Comment:
This is an unusual approach (compare to [cf-units](https://github.com/SciTools/cf-units) or [GeoVista](https://github.com/bjlittle/geovista)). If it isn't working for you, there is no problem with keeping `tox.ini`.